x-pack/plugins/apm/public/components/shared/suggestions_select/index.tsx

```diff
@@ -91,17 +95,20 @@ export function SuggestionsSelect({
   return (
     <EuiComboBox
       async={true}
-      compressed={true}
+      compressed={false}
```

Comment:
Do you think it's better for the alerting field to not be compressed?

Reply:
in favour of consistency I think yes. 

<img width="845" alt="Screenshot 2022-02-03 at 10 36 59" src="https://user-images.githubusercontent.com/3369346/152318423-d18c7acc-1c0d-45c0-9b26-1374a9b22d68.png">

<img width="845" alt="Screenshot 2022-02-03 at 10 37 16" src="https://user-images.githubusercontent.com/3369346/152318399-8dbaea92-04ea-41f2-96f5-6f3db35ea86c.png">